source/developers/license.markdown

```diff
@@ -183,18 +183,8 @@ The Home Assistant source code is released under the following license.
 
    END OF TERMS AND CONDITIONS
 
-   APPENDIX: How to apply the Apache License to your work.
-
-      To apply the Apache License to your work, attach the following
-      boilerplate notice, with the fields enclosed by brackets "{}"
-      replaced with your own identifying information. (Don't include
-      the brackets!)  The text should be enclosed in the appropriate
-      comment syntax for the file format. We also recommend that a
-      file or class name and description of purpose be included on the
-      same "printed page" as the copyright notice for easier
-      identification within third-party archives.
-
-   Copyright {yyyy} {name of copyright owner}
+
+   Copyright 2022 Home Assistant
```

Comment:
2022 seems incorrect in this case?

Reply:
Copyright is usually the current year and updates each year. To list an older date may lead some to conclude that the copyright has expired.
See the bottom of this page, and just about any other website for their copyright notice.

Reply:
> Copyright is usually the current year and updates each year.

That does apply to a general copyright message on a website, but I don't think that applies to a license. Can you point me to a resource that explains it?

Reply:
Seems there are a number of opinions. Some sources say the current year, or the date the code was last updated; whilst others say a span of years (2015-2022 for example).
2022 looks right, but based on research, so would 2015 (Or whatever date the code was first released) either way the duration of copyright is very long and reaches after death, so perhaps it does not matter. Covering bases I guess by going (from-to) rather than a single date.

This is a good read:
[https://softwareengineering.stackexchange.com/questions/130478/what-copyright-date-for-an-update-to-an-open-source-project-from-last-year](url)
And this
[https://opensource.stackexchange.com/questions/2171/refreshing-shared-copyright-dates-in-apache-2-0-licensed-code](url)